### DISCUSSION TITLE
PDFでPDFの表示を貼り込もうとしたときに表紙画像がずれる

### DISCUSSION THREAD

**Discussion by huideyeren** [kmuto/review#1787] [Q&A] [answered]
PDF生成時に生成されたPDFの表紙がずれてしまっています。
これを直す方法はございますでしょうか？

このように表示されます。
<img width="418" alt="スクリーンショット 2022-01-12 12 53 49" src="https://user-images.githubusercontent.com/1094632/149061241-d56fc0c8-ad76-4003-b840-faa68398cad0.png">

なお、リポジトリはこちらです。
https://github.com/huidetang/XiandaiFenshuVol3

**Comment by kmuto** [comment]
sty/review-custom.styのカスタム設定の中で壊れているようです。
試しにreviewmacro.styからreview-custom.styをコメントアウトおよびcatalog.ymlをpreface.reだけにし、Re:VIEW masterブランチからreview-updateした上で実行してみると正常な配置&フィットされました。

[XiandaiFenshuVol3.pdf](https://github.com/kmuto/review/files/7855478/XiandaiFenshuVol3.pdf)

明日時間がとれたらsty/review-custom.styの中をウォッチしてみますね。

**Comment by kmuto** [comment, marked as answer]
XiandaiFenshuVol3プロジェクトではstyがRe:VIEW 5.2のものになっていて、全面に貼るための`\includefullpagegraphics`命令が縦書き非対応です。ずれるのはそれが理由です。

リリース済みのRe:VIEW 5.3で対応しています。
https://review-knowledge-ja.readthedocs.io/ja/latest/releases/review530.html#48e98636215297755f368fa47de6300c

review-updateを行い、review-custom.styから「% 表紙90°回転？」の一連のブロックを削除することで、手元のRe:VIEW 5.3で正常にビルドおよび表紙全面配置ができました。

**Reply by huideyeren** [reply]
解析ありがとうございます。

確かに、 `review-update` を忘れていたのが原因でしたね。
手元のは編集だけしかできないので、環境のアップグレードに関しては今夜帰宅次第対応してみます。

**Reply by huideyeren** [reply]
無事修正完了しました。
ありがとうございます。
